src/components/ChatLog.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css'
+
+const ChatLog = (props) => {
+    const getChat = props.entries.map((chat) => {
+        return ( 
+            <ChatEntry id={chat.id} 
+            sender={chat.sender} 
+            body={chat.body}
+            timeStamp={chat.timeStamp}
+            liked={chat.liked}
+            onClick={props.onClick}
+            key={chat.id}
+            />
+        )
+    })
+    return <div className='chat-log'>{getChat}</div>
```

Comment:
I'm seeing some `divs` for component wrappers that we could replace with more semantic HTML. I'd consider swapping out the enclosing `div` here for something like a `ul` to better represent that this is a list of items being displayed. We could also swap out the `div` that wraps the ChatEntry components to be a `li`; together these changes could help increase accessibility of the page overall.